plugin/completion.py

```diff
@@ -181,22 +282,23 @@ def format_completion(self, item: dict, can_resolve_completion_items: bool) -> s
 
         lsp_label = item["label"]
         lsp_filter_text = item.get("filterText")
-        lsp_detail = item.get("detail") or ""
-        lsp_detail = html.escape(lsp_detail.replace('\n', ' '))
-
-        if can_resolve_completion_items or "documentation" in item:
-            doc_link = '<a href="subl:lsp_resolve_docs">Documentation</a>'
-        else:
-            doc_link = ''
+        lsp_detail = html.escape(item.get("detail", "").replace('\n', ' '))
 
         if lsp_filter_text and lsp_filter_text != lsp_label:
             st_trigger = lsp_filter_text
             st_annotation = lsp_label
-            st_details = '{} {}'.format(doc_link, lsp_detail) if doc_link else lsp_detail
         else:
             st_trigger = lsp_label
-            st_annotation = lsp_detail
-            st_details = doc_link
+            st_annotation = ""
+
+        st_details = ""
+        if can_resolve_completion_items or "documentation" in item:
```

Comment:
It turns out that some language servers send completion items with `{..., "documentation": null, ...}`. We should check this instead:

```
if can_resolve_completion_items or item.get("documentation"):
```

Reply:
Aren't `if can_resolve_completion_items or "documentation" in item:` and `if can_resolve_completion_items or item.get("documentation"):` the same condition? :)

Reply:
It's not the same:

```
>>> f=lambda o: ('x' in o, bool(o.get('x')))
>>> f({})
(False, False)
>>> f({'x': None})
(True, False)
>>> f({'x': ''})
(True, False)
>>> f({'x': 'y'})
(True, True)
```

Reply:
Thanks for the explanation :)
I changed that to `.get('documentaion')`